src/jit/morph.cpp

```diff
@@ -7722,7 +7722,7 @@ GenTreePtr          Compiler::fgMorphCall(GenTreeCall* call)
     {       
         GenTreePtr value = gtArgEntryByArgNum(call, 2)->node;
 
-        if (value->OperGet() == GT_CNS_INT && value->AsIntConCommon()->IconValue() == 0)
+        if (value->IsZero())
```

Comment:
It might be nice to have an `assert(value->OperGet() == GT_CNS_INT)` for the benefit of future readers.